src/main/cpp/subsys/SwerveChassis.cpp

```diff
@@ -467,12 +477,20 @@ void SwerveChassis::AdjustRotToMaintainHeading
 }
 
 void SwerveChassis::AdjustRotToPointTowardGoal
-(
-    units::radians_per_second_t& rot 
+(   
+    Pose2d                     robotPose,
+    Pose2d                     goalPose, 
+    units::meters_per_second_t &xSpeed,
+    units::meters_per_second_t &ySpeed,
+    units::radians_per_second_t &rot     
 )
 {
-    auto myPose = GetPose();
-
+    Pose2d myPose = robotPose;
+    Pose2d targetPose = goalPose;
+    double targetDistance = 90;
+    auto currDistance = m_limelight->EstimateTargetDistance();
+    double error = targetDistance - currDistance.to<double>();

```

Comment:
Why not use the units instead of moving everything to double?  If one distance is in inches and another is not, we won't have consistent values.

Reply:
I have never worked with units till now, so again as my first comment above. Please feel free to update it. :)